dispatcher/MongoConnect.py

```diff
@@ -224,7 +224,10 @@ def aggregate_status(self):
             mode = modes[0]
             run_num = run_nums[0]
             if not _all(modes, mode) or not _all(run_nums, run_num):
+                # Someone did not stop the unlinked (modes)
                 self.logger.error(f'No quorum? {modes}, {run_nums}')
+                for _mode, _run in zip(modes, run_nums):
+                    self.set_stop_time(_run, _mode, force=False)
```

Comment:
I'm not actually sure if this would work or if we'd first need send the stop command before we can actually do such a patch of the endtime

Reply:
That won't work because that's the wrong signature of that function, and the stop command has to be sent first